framework/test/mocha/functional/http/get/voters.js

```diff
@@ -46,57 +46,45 @@ describe('GET /api/voters', () => {
 		expect(res.body.data.voters).to.be.empty;
 	}
 
+	function expectInvalidParamsResponse(res) {
```

Comment:
Although I understand the reasoning behind this change as we're going to move to Jest eventually all tests need to be atomic so I would revert this change so each tests is independent. Even though we could potentially be sure that the assertions will be the same; in the future we might need different ones so I would keep them on each tests instead of using a helper.

Reply:
Oh dear 👍